spec/features/food_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Food', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Username')
+    @food = Food.create(name: 'f_name', measurement_unit: 'kg', quantity: 30, price: 10, user: @user)
+  end
+
+  describe 'index page' do
+    before(:each) do
+      visit '/foods'
+    end
+
+    it 'I can see the foods name ' do
+      expect(page).to have_content 'Name'
+    end
+    it 'I can see the foods quantity' do
+      expect(page).to have_content 'Quantity'
+    end
+    it 'I can see the food measurement unit' do
+      expect(page).to have_content 'Measurement Unit'
+    end
+    it 'I can see the food price' do
+      expect(page).to have_content 'Price'
+    end
```

Comment:
- [ ] These tests are failing as can be seen in the attached image below 👇 Please try to fix them 👍 
![image](https://user-images.githubusercontent.com/46839548/202440868-bea367d7-b619-460b-941f-d1241f193a2f.png)